File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        d = [0,0,0]
        for num in nums:
            d[num] += 1
        idx = 0
        for num, c in enumerate(d):
            for _ in range(c):
                nums[idx] = num
                idx += 1
        
            

File: 0075-sort-colors/0075-sort-colors.js
/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */
var sortColors = function(nums) {
    let isSorted = false;
    while(!isSorted){
        isSorted = true;
        for(let i = 0; i < nums.length - 1; i++){
            if(nums[i] > nums[i + 1]){
                isSorted = false;
                [nums[i], nums[i + 1]] = [nums[i + 1], nums[i]];
            }
        }
    }
};